app/models/chargeback_rate_detail.rb

```diff
@@ -16,11 +16,18 @@ def find_rate(value)
     fixed_rate = 0.0
     variable_rate = 0.0
     chargeback_tiers.each do |tier|
-      next if value < rate_adjustment(tier.start)
-      next if value >= rate_adjustment(tier.finish)
-      fixed_rate = tier.fixed_rate
-      variable_rate = tier.variable_rate
-      break
+      if tier.starts_with_zero?
```

Comment:
@lpichler I want to do this for all cases. I just want to redefine the limits of the tiers intervals: [a,b) => (a,b]. But, in this case, the first interval must also include the 0 => [0,b]